### PR TITLE
chore(docs): update bootstrap documentation

### PR DESCRIPTION
**Pull request by delasteve**

### CODE REVIEW

File: docs/documentation/stories/include-bootstrap.md [hunk 1]
```diff
@@ -3,65 +3,119 @@
...
+```sass
+$icon-font-path: '../node_modules/bootstrap-sass/assets/fonts/bootstrap/';
 ```
+
+In `styles.scss` add the following:
+
+```sass
+// version 3
+@import '_variables';
+@import '../node_modules/bootstrap-sass/assets/stylesheets/_bootstrap.scss';
```
Comment (Brocco) [inline docs/documentation/stories/include-bootstrap.md]: Remove the file extension here.

Comment (delasteve) [inline docs/documentation/stories/include-bootstrap.md]: Good catch. Fixed. 😄

File: docs/documentation/stories/include-bootstrap.md [hunk 2]
```diff
@@ -3,65 +3,119 @@
...
 
+```sass
+$icon-font-path: '../node_modules/bootstrap-sass/assets/fonts/bootstrap/';
 ```
+
+In `styles.scss` add the following:
+
+```sass
+// version 3
+@import '_variables';
```
Comment (cacogr) [inline docs/documentation/stories/include-bootstrap.md]: http://sass-lang.com/documentation/file.SASS_REFERENCE.html#partials

underscore not needed for `@import`

Comment (delasteve) [inline docs/documentation/stories/include-bootstrap.md]: Fixed. Thanks!

**Pull request closed without merging**